hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/receiver/kafka/ConsumerReceiverParameters.java

```diff
@@ -0,0 +1,37 @@
+package pl.allegro.tech.hermes.consumers.consumer.receiver.kafka;
+
+public class ConsumerReceiverParameters {
```

Comment:
I would call this class `KafkaReceiverParameters` as it is used only by `KafkaSingleThreadedMessageReceiver`.